helm/external-secrets/values.yaml

```diff
@@ -1,6 +1,6 @@
 giantswarm:
   images:
-    registry: docker.io
+    repository: docker.io
```

Comment:
It *might* be risky to change this field name if there is no particular reason for it. E.g. if there is an override for it in MCF.

Reply:
Very good point. I changed it to adopt the upstream changes to this value. I've reverted it.

https://github.com/giantswarm/external-secrets/blob/3106466b773a3f6b4173d48751249bf7b1972fe5/helm/external-secrets/templates/deployment.yaml#L47
&& (i hope everywhere else)
https://github.com/giantswarm/external-secrets/blob/3106466b773a3f6b4173d48751249bf7b1972fe5/helm/external-secrets/values.yaml#L3